MapboxNavigation/CarPayCompassView.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+import Mapbox
+
+@objc(MBCarPlayCompassView)
+open class CarPlayCompassView: FloatingButton {
```

Comment:
A circular button on the map may look too much like a tappable button. I suppose that’s why other navigation applications give the compass a rounded rectangle shape, similar to the panels that are overlaid atop the map.

Reply:
Absolutely agree. This should probably be a StylableView or a plain UIView.